app/views/foreman_leapp/job_templates/upgrade.erb

```diff
@@ -9,8 +9,9 @@ template_inputs:
 - name: reboot
```

Comment:
I know this is unrelated, but could you please also rename this to `Reboot`? so it's consistent with the rest of the form? This will also need a change `input('Reboot')`